index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-module.exports = (input, needle, replacement) => {
+module.exports = (input, needle, replacement, fromIndex) => {
```

Comment:
Can you make it an options object instead of arguments? There are too many arguments know which makes the code hard to read. See https://github.com/sindresorhus/boxen/blob/b2378fae5faa3b5e4fe942a742165627b1c200a8/index.js#L66-L72 for how to do it.

And how I usually document it: https://github.com/sindresorhus/boxen#boxeninput-options